src/dhcp6relay/src/main.cpp

```diff
@@ -0,0 +1,16 @@
+#include <stdlib.h>
+#include "configInterface.h"
+
+int main(int argc, char *argv[]) {
+    try {
+        arg_config context;
```

Comment:
is `context` a per vlan struct? How are we handling multi-vlan case?

Reply:
Updated code to support multi-vlan